### ISSUE TITLE
Offboard dracut from kernel dot org infrastructure.

### ISSUE THREAD

**Issue by johannbg** [dracutdevs/dracut#1837]
Consolidate all dracut's resources to a single landing space here on github.

- [x] Move wiki from the kernel.org hosted [wiki](https://dracut.wiki.kernel.org/index.php/Main_Page) to github's [wiki](https://github.com/dracutdevs/dracut/wiki) ( Done me and apparently Harald as well )
- [x] Add a redirect on kernels wiki page to githubs ( Done Harald )
- [x] Send [announcement](https://www.spinics.net/lists/linux-initramfs/msg04912.html) on linux-initramfs mailings about these changes ( Done Harald )
- [ ] Rewrite the repo's README to reflect those changes.]

**Comment by johannbg** [comment]
Correct Tags should be signed which is what I initally did but since Harald was going to sign the release tarball he was planning to upload to kernel.org I backed out of it to avoid confusion ( my signature on the tag, his on the tarball ) which warranty your concern.

That said I will be simplifying the release generation for us all the way down to a single signed git tag ( read as no more signed assets .tarballs or otherwise ) and hopefully automate it in the long run so it will not be subjected to single points of failure.

That said tag creation is currently locked to myself, Lukáš and Harald and or any person we chose to grant privileges to create git tags and in this day and age it should be more than sufficient for downstream consumer of upstreams hosted in git to verify the code by cloning the repo and checking tag, that's it and that's what downstream consumers of upstreams should be using when building downstream systems as opposed to use or rely upon assets like tars which aren't reproducible unless you stick to one specific implementation in one version which makes such solutions less secure than simply using and relying on signed git tags.
